test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests_CSharp.cs

```diff
@@ -795,6 +795,87 @@ public async Task HttpTrigger_Identities_Succeeds()
             }
         }
 
+        [Fact]
+        public async Task HttpTrigger_StaticWebApps_NoClientPrincipalHeader_Identities_Succeeds()
```

Comment:
Let's add a test for expected behavior when both EasyAuth is enabled and it is a static web app, considering BYO functions is coming along in a bit.